cloud-controller-manager/do/firewall_controller_print.go

```diff
@@ -0,0 +1,96 @@
+/*
+Copyright 2020 DigitalOcean
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package do
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/digitalocean/godo"
+)
+
+func printRelevantFirewallParts(fw *godo.Firewall) string {
+	if fw == nil {
+		return "<n/a>"
```

Comment:
nit: do you think we can make `"<n/a>"` a constant, so that the reader could have more context regarding the relevancy of this string. Or a comment as to why it has to be `"<n/a>"`.